ironfish-cli/src/commands/repl.ts

```diff
@@ -1,6 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import * as IronfishSDK from '@ironfish/sdk'
```

Comment:
Can we just import as `sdk`?

Reply:
Ironfish repl implies it is ironfish, no? Also this is consistent with how it is used in the node code.